leetcode/src/23.c

```diff
@@ -0,0 +1,67 @@
+/**
+ * Definition for singly-linked list.
+ * struct ListNode {
+ *     int val;
+ *     struct ListNode *next;
+ * };
+ * 
+ * Algorithm merges pairwaise lists by divide on conquer.
```

Comment:
```suggestion
 * Algorithm merges pairwise lists by divide and conquer.
```

Reply:
Just fixed the typos. Regarding "enable actions", I think it is enabled, at least it was clicked when I created PR. Also fixed merge conflict for the #1222 PR.

Reply:
Oh, I am sure actions are enabled at the repository level, not when you open PRs, I didn't see any workflow run [here](https://github.com/mindaugl/C/actions). Also #1221 is still out-of-date, you might also want to sync your fork's master branch and delete branches whose PR has already been merged

Reply:
I just enabled workflow runs and merged #1221